bin/config.js

```diff
@@ -1,41 +1,15 @@
 const path = require('path')
 
+const merge = require('deepmerge')
 const { cosmiconfig } = require('cosmiconfig')
 
-const defaultConfig = {
-  icons: [],
-
-  output: {
-    path: path.resolve(__dirname, '../lib'),
-    publicPath: '/',
-    icons: {
-      files: null,
-      sprite: null,
-    },
-  },
-
-  svgo: {
-    plugins: {
-      removeViewBox: false,
-      removeDimensions: true,
-    },
-  },
-}
+const defaultConfig = require('../design.config')
```

Comment:
I still do not support this. Please, restore previous lines and add it as another config into merging.

This will keep the config in place, so you do not have to go to another file to take a look at defaults (even if it's easy with IDE) and will keep your default design config more simple.

I expect this file changing rarely in comparison to `design.config.js` in the project root. It's also safer to keep that config as minimal as possible to prevent mistakes (even if we sure what we do).

Reply:
If it makes you happy why not. I don't see much difference.

Reply:
I was thinking about it a bit more. It comes from the single responsibility principle (SRP). Let's imagine we have many developers of the library. Some of them are designers, who work only on the components and some are engineers who work on the automation part. Engineers don't need to add icons but provide the API to designers who add the components and the icons. The designers, at the same time, should not care about the defaults if there is anything missing but only want to have clear and simple config.

Reply:
All right that makes sense. Thanks for explanation. In that case we shouldn't allow to override svg config at all, should we? I mean it's really minor and it can stay as is.

Reply:
Yes, this assumption seems to be correct but I would keep this possibility at the moment as it's minor 🙂

Maybe we will need it in the future.